### PR TITLE
Add "Remember Me" Functionality to API Login

### PR DESCRIPTION
**Pull request by Niustein**
Cookies implemented for 1 year

### CODE REVIEW

File: routes/api/auth/index.js [hunk 1]
```diff
@@ -38,12 +38,21 @@ router.post('/', async (req, res) => {
     if (!valid) {
         return res.status(400).json({error: 'Invalid password'});
     }
+    
+    const rememberMe = req.body.rememberMe;
+    
+    if(rememberMe){
```
Comment (Step7750) [inline routes/api/auth/index.js]: I think we should make this a bit more strict with `if (rememberMe === true)`.

File: routes/api/auth/index.js [hunk 2]
```diff
@@ -38,12 +38,21 @@ router.post('/', async (req, res) => {
...
+        // Sets cookie length to 1 year
+        req.session.cookie.maxAge = 365 * 24 * 60 * 60 * 1000;
+        req.session.rolling = true;
```
Comment (Step7750) [inline routes/api/auth/index.js]: Rolling doesn't seem to actually do anything when set like this, I think we can safely omit this. The config file lets you change whether the sessions are rolling.

**Comment by Step7750** [comment]
Side note: Do you think we should make the cookie period longer? Perhaps 2 years?

**Comment by Niustein** [comment]
After a bit of testing with the cookie, it seems rolling and resave were unnecessary (at the very least to make it function). Still working without them

**Pull request merged**